### PR TITLE
add whiteboard container

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/all-in-one/issues/5063
Needs https://github.com/nextcloud-releases/all-in-one/pull/55

### CODE REVIEW

File: Containers/whiteboard/start.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+if [ -z "$REDIS_DB_INDEX" ]; then
+    REDIS_DB_INDEX=0
+fi
+
+export REDIS_URL="redis://:$REDIS_HOST_PASSWORD@$REDIS_HOST/$REDIS_DB_INDEX"
+
+# Run it
+npm run server:start
+
+exec "$@"
```
Comment (Zoey2936) [inline Containers/whiteboard/start.sh:19]: why duplicate it?

Comment (szaimen) [reply Containers/whiteboard/start.sh:19]: you mean to remove the exec?

File: Containers/whiteboard/start.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    echo "Waiting for redis to start..."
+    sleep 5
+done
+
+# Set a default for redis db index
+if [ -z "$REDIS_DB_INDEX" ]; then
+    REDIS_DB_INDEX=0
+fi
+
+export REDIS_URL="redis://:$REDIS_HOST_PASSWORD@$REDIS_HOST/$REDIS_DB_INDEX"
```
Comment (szaimen) [inline Containers/whiteboard/start.sh:14]: The problem is that we need to build the redis_url like this...

File: Containers/whiteboard/start.sh (hunk 1 above)
Comment (Zoey2936) [reply Containers/whiteboard/start.sh:19]: either exec or npm run

**Comment by egelhaus** [comment]
@szaimen I have an Question, can I already install the Mastercontainer with the Whiteboard Funtion already? When yes, how?

**Comment by szaimen** [comment]
> @szaimen I have an Question, can I already install the Mastercontainer with the Whiteboard Funtion already? When yes, how?

No, currently not but it will be included in the next AIO update.

**Comment by szaimen** [comment]
This is now released with v9.6.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Pull request merged**